diff --git a/node_modules/@chunge16/vitepress-blogs-theme/src/theme/components/VPBTagIcon.vue b/node_modules/@chunge16/vitepress-blogs-theme/src/theme/components/VPBTagIcon.vue
index b74cdfc..2037cfd 100755
--- a/node_modules/@chunge16/vitepress-blogs-theme/src/theme/components/VPBTagIcon.vue
+++ b/node_modules/@chunge16/vitepress-blogs-theme/src/theme/components/VPBTagIcon.vue
@@ -6,8 +6,6 @@ import { useData } from 'vitepress';
 
 const { theme } = useData();
 
-console.log("theme", theme?.blog);
-
 </script>
 
 <template>
